include/ebpf_structs.h

```diff
@@ -13,7 +13,9 @@ typedef enum bpf_map_type
 {
     BPF_MAP_TYPE_UNSPECIFIED = 0, ///< Unspecified map type.
     BPF_MAP_TYPE_HASH = 1,        ///< Hash table.
-    BPF_MAP_TYPE_ARRAY = 2,       ///< Array, wehere the map key is the array index.
+    BPF_MAP_TYPE_ARRAY = 2,       ///< Array, where the map key is the array index.
+    BPF_MAP_TYPE_PROG_ARRAY =
```

Comment:
Do these IDs need to match the ones on Linux? Or are we assuming that moving the eBPF program from Linux -> Windows always requires generating a new eBFP byte code?